cli/js/make_temp_test.ts

```diff
@@ -26,6 +26,19 @@ unitTest({ perms: { write: true } }, function makeTempDirSyncSuccess(): void {
   assert(err instanceof Deno.errors.NotFound);
 });
 
+unitTest(
+  { perms: { read: true, write: true } },
+  function makeTempDirSyncMode(): void {
+    const path = Deno.makeTempDirSync({ mode: 0o737 });
+    const pathInfo = Deno.statSync(path);
+    if (pathInfo.mode !== null) {
+      // Skip windows
+      // assertEquals(pathInfo.mode, 0o737 & ~Deno.umask());
+      assertEquals(pathInfo.mode & 0o777, 0o715); // assume umask 0o022
+    }
```

Comment:
You should use `if (Deno.build.os == "win")` and test accordingly. The way this is now, it could be that Linux returns null for mode, and the test would pass. Also please don't leave commented out code.

Reply:
Ok, I was following the lead of the existing tests for mkdir. Happy to make all these checks more explicit.

Happy also to delete the commented code, but which test behavior do you prefer: just assume the umask is 0o022 (as is true on the test machines)? Or explicitly query the umask? (Then these are less "unit" tests.)

Reply:
I think explicitly querying sounds better to me.